docs/docs/integrations/providers/mlflow_tracking.ipynb

```diff
@@ -47,142 +57,498 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9011db62",
+   "metadata": {},
+   "source": [
+    "### Invoke the Logged Agent\n",
+    "\n",
+    "Now load the agent back and invoke it. There are two ways to load the model"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 53,
+   "id": "b634b69d",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Downloading artifacts: 100%|██████████| 10/10 [00:00<00:00, 331.57it/s]\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "['It would take approximately 1194.5 hours to drive to the Moon with an F1 racing car.']"
+      ]
+     },
+     "execution_count": 53,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Let's turn on the autologging with default configuration, so we can see the trace for the agent invocation.\n",
+    "mlflow.langchain.autolog()\n",
+    "\n",
+    "# Load the model back\n",
+    "agent = mlflow.pyfunc.load_model(info.model_uri)\n",
+    "\n",
+    "# Invoke\n",
+    "agent.predict(question)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "30bf6133",
+   "metadata": {},
+   "source": [
+    "Navigate to the **\"Traces\"** tab in the experiment and click the request ID link of the first row. The trace visualizes how the agent operate multiple tasks within the single prediction call:\n",
+    "1. Determine what subtasks are required to answer the questions.\n",
+    "2. Search for the speed of an F1 racing car.\n",
+    "3. Search for the distance from Earth to Moon.\n",
+    "4. Compute the division using LLM."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cbd10f34",
+   "metadata": {},
+   "source": [
+    "## Scenario 3. Using MLflow Callbacks\n",
+    "\n",
+    "**MLflow Callbacks** provide a semi-automated way to track your LangChain application in MLflow. There are two primary callbacks available:\n",
+    "\n",
+    "1. **`MlflowLangchainTracer`:** Primarily used for generating traces, available in `mlflow >= 2.14.0`.\n",
+    "2. **`MLflowCallbackHandler`:** Logs metrics and artifacts to the MLflow tracking server."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d013d309",
+   "metadata": {},
+   "source": [
+    "### Passing MlflowLangchainTracer\n",
+    "\n",
+    "When the `MlflowLangchainTracer` instance is passed to the `invoke` method of the chain or agent, MLflow will automatically generate a trace for the call stack and log it to the MLflow tracking server.  The outcome is exactly same as `mlflow.langchain.autolog()`, but this is particularly useful when you want to only trace specific invocation. Autologging is applied to all invocation in the same notebook/script, on the other hand."
```

Comment:
Should we list the additional supported invocation methods and mention that not everything is guaranteed to be compatible?

Reply:
Make sense, will test with other [Runnable interfaces](https://python.langchain.com/v0.1/docs/expression_language/interface/) and add notes.